mu4e/mu4e-main.el

```diff
@@ -228,9 +228,12 @@ clicked."
            when (not (and mu4e-main-hide-fully-read (eq unread 0)))
            concat (concat
                    ;; menu entry
-                   (mu4e--main-action-str
-                    (concat "\t* [j" key "] " name)
-                    (concat "j" key))
```

Comment:
Should wrap in `mu4e--main-action-str` (since at the least it'll make it look like the rest of mu4e's maildir shortcuts that are specified manually with `mu4e-maildir-shortcuts`)